package.json

```diff
@@ -95,17 +94,14 @@
     "jest": "^29.6.1",
     "jest-environment-jsdom": "^29.6.1",
     "js-yaml": "^4.1.0",
-    "lighthouse": "10.0.1",
     "prettier": "2.8.4",
-    "puppeteer": "19.4.1",
```

Comment:
With this removal the devcontainer actually works, as its not trying (and failing) to install chrome anymore.